asapdiscovery-docking/asapdiscovery/docking/openeye.py

```diff
@@ -297,12 +315,18 @@ def _dock(
                             pose_res, retcode = self.run_oe_posit_docking(
                                 opts, pose_res, dus, lig_oemol, self.num_poses
                             )
-
                     if retcode == oedocking.OEDockingReturnCode_Success:
-                        for result in pose_res.GetSinglePoseResults():
+                        input_pairs = []
+                        posed_ligands = []
+                        for i, result in enumerate(pose_res.GetSinglePoseResults()):
                             posed_mol = result.GetPose()
                             prob = result.GetProbability()
 
+                            # clear SD data for uniformity of results
+                            from asapdiscovery.data.backend.openeye import clear_SD_data
+
+                            clear_SD_data(posed_mol)
```

Comment:
We need these tags to preserve postera metadata in some cases.

Reply:
I still think we should just leave this out, no need to clear for no reason. All the tag info should be available regardless.